CHANGELOG.md

```diff
@@ -27,7 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [v3.14.2] - 2022-08-29
 ### :bug: Bug Fixes
 - [`20cdb44`](https://github.com/ietf-tools/xml2rfc/commit/20cdb4460436a7edbebc9807d1c3305e90d0269d) - Fix odd page break inside rows in PDF output *(PR [#879](https://github.com/ietf-tools/xml2rfc/pull/879) by [@kesara](https://github.com/kesara))*
-- [`2c9dfaf`](https://github.com/ietf-tools/xml2rfc/commit/2c9dfafa0f4c32399f9b8e31ef9926c3e5ea7b65) - Return orgnization for orgnization only contacts *(PR [#837](https://github.com/ietf-tools/xml2rfc/pull/837) by [@kesara](https://github.com/kesara))*
+- [`2c9dfaf`](https://github.com/ietf-tools/xml2rfc/commit/2c9dfafa0f4c32399f9b8e31ef9926c3e5ea7b65) - Return organization for organization only contacts *(PR [#837](https://github.com/ietf-tools/xml2rfc/pull/837) by [@kesara](https://github.com/kesara))*
```

Comment:
There's an argument to be made (point raised in offline discussion with Kesara) that this file is history that should not be changed. 

I'm don't think there's any confusion that would be introduced by the changes after this line, since they are original text contributions that appear nowhere else. This line, however, is rendered by an automaton, and the spelling being corrected exists as part of a git commit. While searches against such a commit should be by hash, there is potential for confusion if the rest of the string doesnt match.

As such, I suggest reverting the change to this line at least. I could be convinced that the right thing to do is leave the file alone.